.github/workflows/automatic-release.yml

```
@@ -7,6 +7,10 @@ on:
         default: 18
         required: false
         type: string
+    secrets:
+      USER_TOKEN:
+        description: Authentication for the automatic release action.
```

Let's be more specific about the token's purpose. It is necessary to allow the `semantic-release-bot` to push to the repository, correct? Then let's write that.
```suggestion
        description: Authentication for write permission for the release bot.
```
I'm not really convinced by the wording, either. Do you have a better suggestion? In any case, please align this string with the one from the documentation.

---

Maybe...?
```suggestion
        description: Authentication token with write permission needed by the release bot.
```